docs/european-user-consent.mdx

```diff
@@ -214,45 +125,16 @@ For example:
 ```js
 import { AdsConsent, AdsConsentDebugGeography } from 'react-native-google-mobile-ads';
 
-await AdsConsent.setDebugGeography(AdsConsentDebugGeography.EEA);
+const consentInfo = await AdsConsent.requestInfoUpdate({
+  debugGeography: AdsConsentDebugGeography.EEA,
+  testDeviceIdentifiers: ['TEST-DEVICE-HASHED-ID'],
+});
 ```
 
-> Always ensure debug information is removed for production apps!
-
-### Forwarding the consent status to ads
-
-Assuming the user is within the EEA and has provided consent, their status needs to be forwarded to every ad request we
-make in our application.
-
-> If the user is within the EEA and has not given consent, do not display AdMob ads (even non-personalized).
-
-Taking a Rewarded Video as an example, we can apply the users consent when our ad is loaded via the `RequestOptions`. For example:
+It is also possible to reset the UMP state for test devices. This does not reset the ATT state though.
```

Comment:
> This does not reset the ATT state though

Hmm - any pointers for inexperienced devs here? How does one reset the ATT state? Maybe a link to an apple doc or similar...